src/main/scala/com.snowplowanalytics.iglu/ctl/commands/TableCheck.scala

```diff
@@ -17,15 +17,13 @@ import java.util.UUID
 import cats.data.{EitherT, NonEmptyList}
 import cats.effect._
 import cats.syntax.show._
-import cats.syntax.either._
 import cats.syntax.eq._
-import cats.Show
-import fs2.Stream
+import cats.{Order, Show}
+import cats.implicits.{catsSyntaxOptionId, toTraverseOps}
```

Comment:
best practice is to import from `syntax` i.e. `cats.syntax.option._, cats.syntax.traverse._`